Add Event role add/remove commands

huang-a opened the pull request:
The code is untested because I don't have Discord API tokens set up. The code should™ work when tested though.

Commit:
Add command to add the Event role to yourself

Untested code

Co-authored-by: Andy Huang <ahuang-noreply@protonmail.com>

Commit:
Add command to remove the Event role from yourself

Untested code

Co-authored-by: Andy Huang <ahuang-noreply@protonmail.com>

Nepgfurmixpro commented:
Everything looks good but “addRole” and “removeRole” has been removed from discord.js and is now “roles.add” and “roles.remove”

Commit:
Fix usage of removed methods

GuildMember.{addRole, removeRole} changed to GuildMember.{roles.add, roles.remove}

Co-authored-by: Andy Huang <ahuang-noreply@protonmail.com>

huang-a commented:
> Everything looks good but “addRole” and “removeRole” has been removed from discord.js and is now “roles.add” and “roles.remove”

Ok, that should be fixed now.
